### PR TITLE
Corrected Typos in Blueprints

### PR DESCRIPTION
**Pull request by VaibhavVasdev**
### Summary of changes
Corrected Typos in Blueprints document
### Preview
n/a
### Related Fly.io community and GitHub links
n/a
### Notes
n/a

### CODE REVIEW

File: blueprints/autostart-internal-apps.html.md [hunk 1]
```diff
@@ -4,17 +4,17 @@ layout: docs
...
+You have a private, or internal, app that communicates only with other apps on your [private network](/docs/networking/private-networking/). This private app might be a database, authentication server, or any other "backend" app that you don't want exposed to the public Internet. You want the app's Machines to stop when they're not serving requests from your other apps, and start again automatically when needed.
 
-To use the Fly Proxy autostart and autostop feature you need to configure services in `fly.toml`, like you would for a public app. But instead of using a public Anycast address, you assign a Flycast address to expose those services only on your private network.
+To use the Fly Proxy autostart and autostop features you need to configure services in `fly.toml`, like you would for a public app. But instead of using a public Anycast address, you assign a Flycast address to expose those services only on your private network.
```
Comment (andie787) [inline blueprints/autostart-internal-apps.html.md:9]: leave this as-is, consider it a single feature!

File: blueprints/going-to-production-with-healthcare-apps.html.md [hunk 2]
```diff
@@ -7,17 +7,17 @@ redirect_from: /docs/blueprints/going-to-production-with-hipaa-apps
...
 
-This blueprint runs a developer or operations engineer through the process of evaluating Fly.io's security for HIPAA healthcare apps, launching a pilot application, signing a BAA, and deploying to production.
+This blueprint runs a developer or operations engineer through the process of evaluating Fly.io's security for HIPAA healthcare apps, launching a pilot application, signing a Business Associate Agreement (BAA), and deploying to production.
 
 ## HIPAA and Fly.io Primer
 
 The Health Insurance Portability and Accountability Act (HIPAA) sets the standard for protecting sensitive patient data. Any company that processes protected health information (PHI) must ensure that all the required physical, network, and process security measures are in place and followed.
 
-Fly.io takes a "principle of least privilege" approach to security. Here are the highlights of what healthcare app developers get out of the box on Fly.io:
+Fly.io adheres to the "principle of least privilege" approach to security. Here are the highlights of what healthcare app developers get out of the box on Fly.io:
```
Comment (andie787) [inline blueprints/going-to-production-with-healthcare-apps.html.md:16]: leave as is

File: blueprints/going-to-production-with-healthcare-apps.html.md [hunk 3]
```diff
@@ -100,17 +100,17 @@ Your application will automatically be deployed to the regions specified and com
...
+Fly.io's [secrets manager stores](https://fly.io/docs/reference/secrets/) your application's API credentials and secrets in a secure vault. The contents remain encrypted and are only available from running Machine instances as environment variables.
 
 ### Talk to our solution architects if you have questions
 
 If at any time during your evaluation of Fly you have questions about the infrastructure, security, or service terms you can ask in the community forums or [email Fly.io with your questions](mailto:sales@fly.io).
 
 ## Deploy to production
 
-Once you've evaluated Fly.io and have a BAA, it's time to go to production. The remainder of this guide will walk you through how to setup an isolated production environment, control access, and deploy your application.
+Once you've evaluated Fly.io and have a BAA, it's time to go to production. The remainder of this guide will walk you through setting up an isolated production environment, controlling access, and deploying your application.
```
Comment (andie787) [inline blueprints/going-to-production-with-healthcare-apps.html.md:111]: curious why you suggest this change? I would leave as-is

File: blueprints/going-to-production-with-healthcare-apps.html.md [hunk 4]
```diff
@@ -100,17 +100,17 @@ Your application will automatically be deployed to the regions specified and com
...
 
-### Sign a Business Associate Agreement (BAA)
```
Comment (andie787) [inline blueprints/going-to-production-with-healthcare-apps.html.md:113]: leave as-is

File: blueprints/going-to-production-with-healthcare-apps.html.md [hunk 5]
```diff
@@ -176,4 +176,4 @@ Finally run `fly deploy --path fly.production.yml` to deploy your application, t
 
 ## Wrap-up
 
-Fly.io is a secure, productive platform for deploying HIPAA-compliant healthcare applications. The platform provides the necessary security controls and features to ensure that PHI is protected in accordance with HIPAA requirements while giving development teams the flexibility to ship often and scale quickly.
+Fly.io is a secure, productive platform for deploying HIPAA-compliant healthcare applications. It provides the necessary security controls and features to ensure that PHI is protected in accordance with HIPAA requirements while giving development teams the flexibility to ship often and scale quickly.
```
Comment (andie787) [inline blueprints/going-to-production-with-healthcare-apps.html.md:179]: leave as-is. it's more clear when the subject of the sentence is named

File: blueprints/resilient-apps-multiple-machines.html.md [hunk 6]
```diff
@@ -94,7 +94,7 @@ To create a standby Machine from a different image:
 fly machine run <image> --standby-for <machine-id>
 ```
 
-## Benefits and cost of multiple Machines
+## Benefits and costs of multiple Machines
```
Comment (andie787) [inline blueprints/resilient-apps-multiple-machines.html.md:97]: leave as-is, since "Cost" is use below and is the intended word here together with "of"

**Review by andie787 — COMMENTED** [review]
thanks for your input. I've left some notes on what I think should stay as -is. In general, it's probably best to fix typos and leave style preferences aside; makes it easier and quicker for us to review and approve.

**Comment by VaibhavVasdev** [comment]
Hi @andie787, thank you for the review. I agree with your comments and I will make the required changes.

Additionally, I will keep them in mind while making further contributions.

File: blueprints/autostart-internal-apps.html.md (hunk 1 above)
Comment (VaibhavVasdev) [reply blueprints/autostart-internal-apps.html.md]: I have made the required change. Thank you for the review.

File: blueprints/going-to-production-with-healthcare-apps.html.md (hunk 2 above)
Comment (VaibhavVasdev) [reply blueprints/going-to-production-with-healthcare-apps.html.md]: I have made the required change. Thank you for the review.

File: blueprints/going-to-production-with-healthcare-apps.html.md (hunk 3 above)
Comment (VaibhavVasdev) [reply blueprints/going-to-production-with-healthcare-apps.html.md]: I have made the required change. Thank you for the review.

File: blueprints/going-to-production-with-healthcare-apps.html.md (hunk 5 above)
Comment (VaibhavVasdev) [reply blueprints/going-to-production-with-healthcare-apps.html.md]: I have made the required change. Thank you for the review.

File: blueprints/resilient-apps-multiple-machines.html.md (hunk 6 above)
Comment (VaibhavVasdev) [reply blueprints/resilient-apps-multiple-machines.html.md]: I have made the required change. Thank you for the review.

File: blueprints/going-to-production-with-healthcare-apps.html.md (hunk 4 above)
Comment (VaibhavVasdev) [reply blueprints/going-to-production-with-healthcare-apps.html.md]: I have made the required change. Thank you for the review.

**Comment by VaibhavVasdev** [comment]
Hi @andie787  I have rectified all the as-is cases. 
Apologies for any inconvenience and I will keep in mind your comments before making any further commits in the future.
Thank you, for the review.

**Pull request merged**